client/js/components/Navigation.jsx

```diff
@@ -3,12 +3,14 @@ import React from 'react';
 
 export default class Navigation extends React.Component {
     render () {
+        var h1 = <h1 onClick={() => window.location.reload()}>Austin Green Map</h1>;
```

Comment:
Yeah as long as h1 isn't modified in the same block, it'd be a great place to use const instead of var. As a rule of thumb, removing state from functions makes them easier to manage. 

https://github.com/airbnb/javascript#2.1